### PR TITLE
Fix order autoupdate and chat

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?
- Removes all uses of `robot.token`, preparing the code for a future movement of token specific data to Slot
- Fixes Order auto-update for both maker and taker
- Fixes chat (both socket and turtle)
- Sometimes the premium input freeze because coordinator's limits were not properly loaded.

## Checklist before merging
- [x] Install [pre-commit](https://pre-commit.com/#installation) and initialize it: `pip install pre-commit`, then `pre-commit install`. Pre-commit installs git hooks that automatically check the codebase. If pre-commit fails when you commit your changes, please fix the problems it points out.

### CODE REVIEW

File: frontend/src/components/TradeBox/EncryptedChat/EncryptedSocketChat/index.tsx [hunk 1]
```diff
@@ -98,13 +103,20 @@ const EncryptedSocketChat: React.FC<Props> = ({
...
+    if (!slot?.token) return;
+
+    const { url, basePath } = federation
+      .getCoordinator(order.shortAlias)
+      .getEndpoint(settings.network, origin, settings.selfhostedClient, hostUrl);
 
     websocketClient
       .open(
-        `ws://${window.location.host}/ws/chat/${orderId}/?token_sha256_hex=${sha256(robot?.token)}`,
+        `${url.replace(/^https?:\/\//, 'ws://') + basePath}/ws/chat/${
```
Comment (KoalaSat) [inline frontend/src/components/TradeBox/EncryptedChat/EncryptedSocketChat/index.tsx:117]: this regexp should work for both http and https

**Comment by Reckless-Satoshi** [comment]
Merging as is for more tests. Please share another 150K invoice for the great working on figuring out where this issues originated and their fix! :D

**Comment by KoalaSat** [comment]
🚀 
`lnbc1500u1pjejkvapp58p8m2tmzegwe4qc2dee2l9satez4vd48nc708y7hmxrgy2e036eqdq5g9kxy7fqd9h8vmmfvdjscqzzsxqyz5vqsp5ajmlsjjt4kq9lrvweqegae0hh33y27jqmzu63v5xy3t72renyjds9qyyssqdla9vk3r7mgdgnpy5jvyj86eg57wrg8xpyknjs3gjaplu3x82qajm5s7kkf4mfjraw2gq6gjly75x84t9upj4a3rgnk5s87rfhxp5lcp8pemtq`

**Comment by Reckless-Satoshi** [comment]
> lnbc1500u1pjejkvapp58p8m2tmzegwe4qc2dee2l9satez4vd48nc708y7hmxrgy2e036eqdq5g9kxy7fqd9h8vmmfvdjscqzzsxqyz5vqsp5ajmlsjjt4kq9lrvweqegae0hh33y27jqmzu63v5xy3t72renyjds9qyyssqdla9vk3r7mgdgnpy5jvyj86eg57wrg8xpyknjs3gjaplu3x82qajm5s7kkf4mfjraw2gq6gjly75x84t9upj4a3rgnk5s87rfhxp5lcp8pemtq

e81d2420725887e57f6b86eda08d388069fca49f660f17b32b9f8e9783370b5f

**Pull request merged**